### ISSUE TITLE
Website is down

### ISSUE THREAD

**Issue by mcherep** [buntine/pioneers#7]
This is not a bug, but the website has been down for a while and I don't know if it's intended.

**Comment by buntine** [comment]
Yep, I need to register the domain name again. I will try and do it tomorrow!

**Comment by mcherep** [comment]
Ping :)
